p4-14/v1.0.4/tex/p4.pt

```diff
@@ -1977,6 +1976,10 @@ The hash value is used to generate a value between \texttt{base} and
 The result can be used as a hash digest of a long key (such as 5-tuple) or as 
 an offset/index to reference a register array. Normal value conversion 
 takes place when setting \texttt{dest} to the result.
+
+If any of the fields in the \texttt{field_list_calc} have a parent
+header that is not valid, then the operation leaves \texttt{dest} with
+an undefined value.
```

Comment:
Here is a quote on this topic from @antoninbas from this PR: https://github.com/p4lang/p4-spec/pull/174

"bmv2 skips invalid fields / headers in calculations, as this has proved very convenient in P4 programs which hash on L4 fields
this is also consistent with deparsing. IMO a field list calculation is just: 1) serializing the fields to a buffer (just like deparsing), and 2) applying a calculation over the buffer."

If people are OK with making this the official defined behavior of the 'modify_field_with_hash_based_offset' primitive, then I propose changing the phrase above:

    "then the operation leaves \texttt{dest} with an undeifned value"

to something like this:

    "then those header fields are left out of the field list for the purposes of calculating the hash value."